examples/composite_dialog/my_app_window/mod.rs

```diff
@@ -0,0 +1,80 @@
+mod imp;
+
+use gtk::{
+    glib::{self, ParamSpec},
```

Comment:
No includes for a one time used type please

Reply:
use statement removed for ParamSpec.